w3-revocations-check.md

```diff
@@ -0,0 +1,185 @@
+# W3 Revocations Check Protocol
+
+## Authors
+
+- [Felipe Forbeck](https://github.com/fforbeck)
+
+## Editors
+
+- [Alan Shaw](https://github.com/alanshaw)
+
+## Abstract
+
+The W3 Revocations Check protocol provides a public API endpoint for checking if specific delegation CIDs have been revoked and retrieving cryptographic proof data for trustless verification. This protocol enables clients to verify the validity of UCAN delegations by checking their revocation status.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+# Introduction
+
+The revocations check protocol allows clients to verify whether UCAN delegations have been explicitly revoked. The protocol provides a simple HTTP GET endpoint that returns either cryptographic proof of revocation or indicates that no revocation exists for the queried delegation CID.
+
+# Capabilities
+
+## Revocation Check
+
+### Endpoint Details
+
+- **Method:** `GET`
+- **Path:** `/revocations/{cid}`
+- **Authentication:** None required (public endpoint)
+- **CDN Cacheable:** Yes (simple GET requests with 200/404 responses)
+
+### Request Format
+
+#### URL Parameters
+
+**`cid`** (required): Delegation CID to check for revocations. Must be a valid IPFS CID string.
+
+#### Constraints
+
+- **CID Format:** Valid IPFS CID string (e.g., `bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi`)
+- **Encoding:** CID will be normalized to string format internally
+
+### Response Format
+
+#### Success Response (200): Delegation Revoked
+
+**Content-Type:** `application/vnd.ipld.car`
+
+Returns a CAR (Content Addressable aRchive) file containing:
+
+- Revocation metadata in `revocations@0.0.1` schema
+- Embedded UCAN proof blocks for trustless verification
+- ETag header for caching optimization
+
+##### CAR File Structure
+
+The CAR file contains a root block with the following structure:
+
+```json
+{
+  "revocations@0.0.1": {
+    "revocations": [
+      {
+        "delegation": { "/": "[delegationCID]" },
+        "scope": "[scopeDID]", 
```

Comment:
I don't think we need to include this, as it is derivable from the cause invocation.

Note: scope is the DID of the authority that issued the revocation - either the issuer or audience of the delegation that is under inspection or one of it's proofs.